lib/ansible/utils/module_docs_fragments/nxos.py

```diff
@@ -73,6 +73,12 @@ class ModuleDocFragment(object):
     required: false
     default: no
     choices: ['yes', 'no']
+  timeout:
+    description:
+      - Specifies idle timeout for the connection.  NX-API can be slow to 
+        return on long-running commands (sh mac, sh bgp, etc).
+    required: false
+    default: 10
```

Comment:
Will need to add a `version_added: 2.3` in here.

Or 2.2. if @privateip is happy to accept and cherry pick this change into `stable-2.2`